### PR TITLE
feat(autoware_accel_brake_map_calibrator): conditional actuation data processing based on source

### PR DESCRIPTION
**Pull request by N-Eiki**
## Description

Fixed to work according to accel_brake_value_source_ as it was not able to separate cases using accel_brake_value_source_.

## Related links
[TIER IV Internal link](https://star4.slack.com/archives/C03QW0GU6P7/p1724295340000889)

## How was this PR tested?
CI 

## Notes for reviewers

None.

## Interface changes

None.

## Effects on system behavior

None.

### CODE REVIEW

File: vehicle/autoware_accel_brake_map_calibrator/src/accel_brake_map_calibrator_node.cpp [hunk 1]
```diff
@@ -267,14 +267,14 @@ bool AccelBrakeMapCalibrator::take_data()
 {
   // take data from subscribers
   // take actuation data
-  ActuationStatusStamped::ConstSharedPtr actuation_status_ptr = actuation_status_sub_.takeData();
-  ActuationCommandStamped::ConstSharedPtr actuation_cmd_ptr = actuation_cmd_sub_.takeData();
-  if (actuation_status_ptr) {
```
Comment (shmpwk) [inline vehicle/autoware_accel_brake_map_calibrator/src/accel_brake_map_calibrator_node.cpp:272]: Don't we need this validation?

**Comment by shmpwk** [comment]
@N-Eiki 
Thank you for your modification!
Could you translate the PR description in English so that English members in AWF can understand this PR.
Also, please set the private link to [TIER IV Internal link](link url) so that the external TIER IV members won't confuse it.

**Comment by N-Eiki** [comment]
I was careless.
I have corrected them.

**Review by shmpwk — APPROVED** [review]
LGTM

**Comment by shmpwk** [comment]
@taikitanaka3 @1222-takeshi @tkimura4 
Give us your approval :pra

**Review by taikitanaka3 — APPROVED** [review]

**Pull request merged**